documentation/meta/project_boards/issues.md

```diff
@@ -0,0 +1,49 @@
+# Issues project: Openverse Backlog
+
+The [Openverse Backlog](https://github.com/orgs/WordPress/projects/75) project
+board tracks all issues through their lifecycle, as they move from creation to
+completion. This board does not track any pull requests, so all workflows for
+this board are tied to events occurring for issues.
+
+## Event automations
+
+### Issue is created
+
+If a new issue is created, it is automatically added to the project board
+provided it does not contain any label with the text "project".
+
+```{note}
+This workflow also sets the Priority custom field in the issue so that we can
+create a kanban-board view based on priority.
+```
+
+- [Custom workflow](https://github.com/WordPress/openverse/blob/main/.github/workflows/new_issues.yml)
+
+### Issue is closed
+
+If an issue is closed, it moves into the "✅ Done" column. This is not affected
+by whether it was closed as resolved or as discarded.
+
+- [Built-in workflow](https://github.com/orgs/WordPress/projects/75/workflows/6899392)
+
+### Issue is reopened
+
+If a previously closed issue is reopened, it goes back to the "📋 Backlog"
+column. That is because it needs to be prioritised again.
```

Comment:
This last sentence makes it sound as if the issue's priority labels are reset, but they stay the same, don't they?

Reply:
By prioritisation, I mean the process by which we move upcoming tickets from the "Backlog" to the "To do" column. Is there a better name for this process?

Reply:
Fixed in b54362b.